### ISSUE TITLE
Ubuntu 16.04 Helix Dockerfile build failure - hash sum mismatch

### ISSUE THREAD

**Issue by mthalman** [dotnet/dotnet-buildtools-prereqs-docker#426]
Attempting to build https://github.com/dotnet/dotnet-buildtools-prereqs-docker/blob/main/src/ubuntu/16.04/helix/arm32v7/Dockerfile results in the following build error:

```
Failed to fetch https://download.mono-project.com/repo/ubuntu/dists/preview-xenial/main/binary-armhf/Packages  Hash Sum mismatch
```

**Comment by MichaelSimons** [comment]
[Triage] @MattGal - Can you please take a look?

**Comment by MattGal** [comment]
This is actually a problem with https://download.mono-project.com/repo/ubuntu/dists/preview-xenial/main/binary-armhf/ ( added I believe to make sure mono libgdiplus testing works as expected).    Simply changing this to `nightly-xenial` away from `preview-xenial` unblocks the build.

I think we should probably solve this either by moving to a different flavor than preview (@directhex is the SME I believe) and/or opening a bug in the appropriate spot in Mono-land.

**Comment by MattGal** [comment]
opened said bug.

**Comment by directhex** [comment]
Move away from preview, or use the origin-download domain. It’s regenerated frequently, but for annoying new security reasons, the CDN cache is no longer flushed when new metadata is pushed.

Sent from Mail for Windows 10

**Comment by MattGal** [comment]
PR merged, addressed the same potential issue in 18.04 at the same time
